.github/workflows/build_demos.yml

```diff
@@ -0,0 +1,50 @@
+name: Demos
+
+on:
+  pull_request:
+    branches: '**'
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: recursive
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.8"
+    - name: python -m pip install --upgrade pip && python -m pip install numpy
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install numpy
+
+    - name: Cache OpenCV
+      id: opencv-cache
+      uses: actions/cache@v2
+      with:
+        path: ${{ github.workspace }}/opencv-install
+        key: ${{ runner.os }}-opencv-cache
+
+    - name: Install OpenCV
+      uses: rayandrews/with-opencv-action@v1
+      with:
+        dir: ${{ github.workspace }}/opencv-install
+        cached: ${{ steps.opencv-cache.outputs.cache-hit }}
+        opencv-version: '4.7.0'
```

Comment:
this case takes 2GB https://github.com/openvinotoolkit/open_model_zoo/actions/caches
looks too much even for OpenCV :)

Reply:
Fixed